File: app/src/main/java/ua/meugen/android/pdfcomposer/model/db/entity/PdfItemEntity.java
package ua.meugen.android.pdfcomposer.model.db.entity;

import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;

import java.util.Date;

@Entity(tableName = "pdfitems")
public class PdfItemEntity {

    @PrimaryKey(autoGenerate = true)
    public int id;
    public String name;
    public String path;
    public long size;
    public Date date;
}

File: app/src/main/java/ua/meugen/android/pdfcomposer/model/db/entity/PdfItemEntity.kt
package ua.meugen.android.pdfcomposer.model.db.entity

import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

import java.util.Date

@Entity(tableName = "pdfitems")
class PdfItemEntity {

    @PrimaryKey(autoGenerate = true)
    var id: Int = 0
    var name: String? = null
    var path: String? = null
    var size: Long = 0
    var date: Date? = null
}
